docs/pages/input.html

```diff
@@ -0,0 +1,135 @@
+                <span class="icon">
+                    <i class="fas fa-bars "></i>
+                </span>
+            </button>
+            <ul class="navbar-menu fs-20">
+                <li class="navbar-item"><a href="../index.html" class="item-link">Docs</a></li>
+                <li class="navbar-item"><a href="https://github.com/rahulmistry751/Kamui" target="_blank" class="item-link"><i class="fab fa-github fa-lg"></i></a></li>
+            </ul>
+        </nav>
+        <main class="main-container">
+            <div class="side-bar">
+                <div class="list-container">
+                    <h3 class="list-title fw-600 fs-20 mgb-16">Getting Started</h3>
+                    <ul class="list">
+                        <li class="list-item"><a href="../index.html" class="item-link">Introduction</a></li>
+                    </ul>
+                </div>
+                <div class="list-container">
+                    <h3 class="list-title fw-600 fs-20 mgb-16">Components</h3>
+                    <ul class="list">
+                        <li class="list-item"><a href="./alert.html" class="item-link">Alert</a></li>
+                        <li class="list-item"><a href="./avatar.html" class="item-link">Avatar</a></li>
+                        <li class="list-item"><a href="./badge.html" class="item-link">Badge</a></li>
+                        <li class="list-item"><a href="./button.html" class="item-link">Button</a></li>
+                        <li class="list-item"><a href="./cards.html" class="item-link">Cards</a></li>
+                        <li class="list-item"><a href="./image.html" class="item-link">Image</a></li>
+                        <li class="list-item"><a href="#" class="item-link">Input</a></li>
+                        <li class="list-item"><a href="./list.html" class="item-link">List</a></li>
+                        <li class="list-item"><a href="./modal.html" class="item-link">Modal</a></li>
+                        <li class="list-item"><a href="./navigation.html" class="item-link">Navigation</a></li>
+                        <li class="list-item"><a href="./utility.html" class="item-link">Utility</a></li>
+                    </ul>
+                </div>
+            </div>
+            <div class="main-content">
+                <h3 class="content-title h3 fw-900">Input</h3>
+                <p class="para lh-24">
+                    Input are used to take user input for the context shown. Input component is mostly used in forms.
+                </p>
+                <hr>
+                <h3 class="content-title h3 fw-700">Input</h3>
+                <div class="example-container fx-c">
+                   <!-- Input -->
+                    <div class="input-container">
+                        <label for="">
+                            Username
+                        </label>
+                        <input type="text" class="input">
```

Comment:
is there any specific reason to not pass any value in the` for=""` attribute ??

To associate the `<label>` with an `<input>` element, you need to give the `<input>` an id attribute. The `<label>` then needs a for attribute whose value is the same as the input's id:
-e.g `<label for="input1">Username</label>
                                                                         <input type="text" class="input">`

Reply:
yeah. I kept it blank deliberately. I thought user could add it's own id name depending on the application.